uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -237,8 +246,8 @@ public void connect(String clientDeviceId, String componentId, String brokerId)
     public void connect(String clientDeviceId, String componentId, String brokerId, String mqttVersion) {
 
         // get address information about broker
-        final List<GGGroup> groups = brokers.get(brokerId);
-        if (groups == null) {
+        final List<MqttBrokerConnection> bc = brokers.get(brokerId);
```

Comment:
Please rename bc to mqttBrokerConnections

Reply:
fixed 8e80651166627c92920165de239a88a247c7d4a5